picky-asn1-x509/src/private_key_info.rs

```diff
@@ -148,6 +176,7 @@ impl ser::Serialize for PrivateKeyValue {
     {
         match self {
             PrivateKeyValue::RSA(rsa) => rsa.serialize(serializer),
+            PrivateKeyValue::EC(ec) => serializer.serialize_newtype_struct(HeaderOnly::<()>::NAME, ec),
```

Comment:
A workaround to serialize `ECPrivateKey` fields only, without a `SEQUENCE` tag.

Reply:
(see above)